app/controllers/orders_controller.rb

```diff
@@ -37,13 +37,19 @@ def create
         project_id: project.id,
         order_id: @order.id,
         quantity: quantity,
-        line_project_cost: project.price * quantity.to_i
+        line_item_cost: project.price * quantity.to_i
```

Comment:
I thought that we were using project cost and not line_item cost? I also don't think we need line_item_cost anymore since we don't have quantity anymore. Our orders only have total_cost now which refers to the aggregate of loan amounts.